presidio-analyzer/tests/conf/test_minimal_registry_conf.yaml

```diff
@@ -0,0 +1,2 @@
+global_regex_flags=26
+recognizers= []
```

Comment:
Let's think what should we do in this case. Load all predefined? or load 0? currently it loads all predefined